src/ga.js

```diff
@@ -179,7 +187,7 @@ function sendBidRequestToGa(bid) {
   if (bid && bid.bidderCode) {
     _analyticsQueue.push(function () {
       _eventCount++;
-      window[_gaGlobal]('send', 'event', _category, 'Requests', bid.bidderCode, 1, _disibleInteraction);
+      window[_gaGlobal]( trackerSend, 'event', _category, 'Requests', bid.bidderCode, 1, _disibleInteraction);
```

Comment:
`trackerSend` is undefined when this function is called, you probably want a local variable for it. Also the test needs to be updated with the exported `trackerSend` -- though I believe that is the only case where the export is needed?